src/synthetic-monitoring/01_analytics_workspace.tf

```diff
@@ -0,0 +1,28 @@
+resource "azurerm_resource_group" "synthetic_rg" {
+  location = var.location
+  name     = "${local.project}-rg"
+}
+
+
+resource "azurerm_log_analytics_workspace" "log_analytics_workspace" {
+  name                = "${local.project}-law"
+  location            = azurerm_resource_group.synthetic_rg.location
+  resource_group_name = azurerm_resource_group.synthetic_rg.name
+  sku                 = var.law_sku
+  retention_in_days   = var.law_retention_in_days
+  daily_quota_gb      = var.law_daily_quota_gb
+
+  tags = var.tags
+}
+
+# Application insights
+resource "azurerm_application_insights" "application_insights" {
```

Comment:
i think is better to create a generic monitoring_tools or something similar, to be able to use one for all